src/components/plots/gridPlot.ts

```diff
@@ -43,28 +48,49 @@ export module Plot {
     }
 
     /**
-     * @param {string} attrToSet One of ["x", "y", "fill"]. If "fill" is used,
+     * @param {string} attrToSet One of ["x", "y", "x2", "y2", "fill"]. If "fill" is used,
```

Comment:
GridPlot only takes x/y/x2/y2. If it were quantitative you would be projecting x/x2, which internally gets converted into an x1